SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1601,3 +1601,43 @@ void USpatialNetDriver::HandleOnConnectionFailed(const FString& Reason)
 	UE_LOG(LogSpatialOSNetDriver, Error, TEXT("Could not connect to SpatialOS. Reason: %s"), *Reason);
 	OnConnectionFailed.Broadcast(Reason);
 }
+
+uint32_t USpatialNetDriver::AddOpCallback(Worker_ComponentId ComponentId, const UserOpCallback& Callback)
```

Comment:
I spoke with @alastairdglennie offline, I think it would be nice if we moved this functionality into the spatial dispatcher so that we can minimize the responsibility of the NetDriver. this would also make it easier to implement tests for this in the future.

@samiwh what's your thoughts?

Reply:
I agree. The net driver should have as little behviour as possible, and this might need more of a dispatcher than it currently has.

Reply:
I also agree.